kong/resource_kong_consumer_plugin_config.go

```diff
@@ -159,7 +159,6 @@ func resourceKongConsumerPluginConfigRead(d *schema.ResourceData, meta interface
 
 	d.Set("consumer_id", idFields.consumerId)
 	d.Set("plugin_name", idFields.pluginName)
-	d.Set("tags", consumerPluginConfig.Tags)
```

Comment:
why is this line being removed? can you pop something in the description to explain please :)